doc/python/simulation.rst

```diff
@@ -135,6 +135,11 @@ over the local and distributed hardware resources (see :ref:`pydomdec`). Then, t
 
         Retrieve a list of sample data associated with the given ``handle``.
         There will be one entry in the list per probe associated with the probe id used when the sampling was set up.
+        E.g. if a probe was placed on a locset describing three positions, the returned list will have three elements.
```

Comment:
Starting a sentence with E.g. looks odd.

**For example**, if a probe... the returned list will **contain** three elements.